### PR TITLE
New plugin Vatin

### PR DESCRIPTION
**Pull request by fansanelli**
Hello,

please find my PR to check if an italian "Partita IVA" is well formed.
The tests are passing:

```
$ nosetests plugins/TagFix_Vatin.py 
.
----------------------------------------------------------------------
Ran 1 test in 0.773s

OK
```

Please let me know what you think
Many thanks
Best Regards,
Francesco

### CODE REVIEW

File: plugins/TagFix_Vatin.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+###########################################################################
+
+from plugins.Plugin import Plugin
+import re
+
+class TagFix_Vatin(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[99999] = {"item": 9999, "level": 3, "tag": ["ref:vatin"], "desc": T_(u"Invalid ref:vatin")}
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Please be more explicit: `T_(u'Invalid value of tag "'ref:vatin"')`

Comment (frodrigo) [reply plugins/TagFix_Vatin.py]: The `tag` cannot be `ref:vatin`. Pick tags from this list http://osmose.openstreetmap.fr/en/api/0.2/meta/tags

File: plugins/TagFix_Vatin.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    def it_vatin(self, vatin):
+        if (len(vatin) != 11):
+            return False
+
+        s = 0
```
Comment (frodrigo) [inline plugins/TagFix_Vatin.py]: Please a comment about this, a link ?

**Comment by fansanelli** [comment]
Dear frodrigo,

I'll try to address you comment sooner than later, but please be aware that this plugin can be extended to any country even I decided to focus on Italy first...
On Italian Wikipedia you can find the regex to match other countries too:
https://it.m.wikipedia.org/wiki/Partita_IVA

**Comment by fansanelli** [comment]
> Please rename the plugin to TagFix_it_Vatin

I decided to keep original name in V2, but to add a comment:

    # XXX Enable more countries
    only_for = ["IT"]

**Comment by frodrigo** [comment]
OK for IT

**Pull request closed without merging**